lib/logstash/filters/translate.rb

```diff
@@ -81,7 +81,12 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
 
   # When using a dictionary file, this setting will indicate how frequently
   # (in seconds) logstash will check the dictionary file for updates.
-  config :refresh_interval, :validate => :number, :default => 300
+  config :refresh_interval, :validate => :number, :default => 5
+  # when using a dictionary file, if this is set to true, the dictionary's
+  # modification time will be checked before reloaded.  If the dictionary
+  # isn't modified, the refresh won't happen.  If you set this to false,
+  # increase the refresh_interval.
+  config :refresh_only_if_modified, :validate => :boolean, :default => true
```

Comment:
I am thinking this behavior should be the default and not exposing a setting for it.

Why would a user ever want to set this to `false` ?